core/services/ccip/orm.go

```diff
@@ -0,0 +1,207 @@
+package ccip
+
+import (
+	"context"
+	"fmt"
+	"math/big"
+	"time"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/sqlutil"
+	"github.com/smartcontractkit/chainlink-common/pkg/types/ccip"
+)
+
+type GasPrice struct {
+	SourceChainSelector uint64
+	GasPrice            *big.Int
+	CreatedAt           time.Time
+}
+
+type GasPriceRow struct {
```

Comment:
Is it possible to use scanner/value types for the fields so that we don't need to have extra types and conversions?

Reply:
replace big.int with assets.wei, found it has existing scanner/valuer impl